src/main/java/cz/cvut/kbss/termit/service/repository/VocabularyRepositoryService.java

```diff
@@ -127,9 +127,9 @@ private void verifyVocabularyImports(Vocabulary update) {
     }
 
     @Override
-    public Vocabulary importVocabulary(String vocabularyIri, MultipartFile file) {
+    public Vocabulary importVocabulary(boolean rename, URI vocabularyIri, MultipartFile file) {
```

Comment:
Here, the parameter order is `rename`, `vocabularyIri` and `file`, but in `SKOSImporter`, the order is `vocabularyIri`, `rename` and `file`-related stuff. IMHO the order of parameters should be consistent.